Link to oss dashboard. (#179)
qnx-ports/build-files: README.md

2. **Host Build:**
   You can build ports directly on your host system.

## Open-Source Dashboard

> All build and test results will be posted to the official Open-Source Dashboard.

Reviewer: Usually would prefer a present tense:

```suggestion
All build and test results are posted to the official Open-Source Dashboard.
```
